### PR TITLE
Polish a few broken things in the sidebar.

### PR DESCRIPTION
**Pull request by jasmussen**
There were some issues with the block sidebar:

- The active style looked a bit overwhelming.
- The button-group control lacked a little style.
- The Width/Height controls for an image were offset.

This PR addresses those.

Before:

Screenshot 2020-03-11 at 10 10 02

After:

Screenshot 2020-03-11 at 10 16 03

The button-group after:

Screenshot 2020-03-11 at 10 24 23

Before:

Screenshot 2020-03-11 at 10 30 55

After:

Screenshot 2020-03-11 at 10 41 46

**Note that I expect all of the above to receive additional polish**. #18667 offers additional avenues to refining how each of these controls look, behave, and are positioned well in the sidebar. I consider this PR a _minimal_ first step towards making it less broken, but not the endgame step.

### CODE REVIEW

File: packages/components/src/base-control/style.scss [hunk 1]
```diff
@@ -20,7 +20,3 @@
 		font-style: italic;
 	}
 }
-
-.components-base-control + .components-base-control {
```
Comment (youknowriad) [inline packages/components/src/base-control/style.scss]: I wonder if we should remove that rule just like that. It has implications on all sidebars (and all other places where we use multiple controls in a row like the entities save modal) (multiple controls in a row) including in third-party blocks.

Comment (youknowriad) [reply packages/components/src/base-control/style.scss]: I'd rather override it personally where we have horizontal controls today.

Comment (jasmussen) [reply packages/components/src/base-control/style.scss]: I responded in https://github.com/WordPress/gutenberg/pull/20608#discussion_r390853392 — best I can tell it's either dead code or ill-advised. In all my web-inspecting, the rule was almost always overridden. 

The way to space base controls, I would suggest, is simple:

```
.components-base-control {
	margin-bottom: $grid-unit-20;
}
```

While you can achieve the same with the following:

```
.components-base-control + .components-base-control {
	margin-top: $grid-unit-20;
}
```

That's a far more roundabout way of doing it. And given margins collapse regardless, the former seems much more the correct way of spacing things.

Comment (youknowriad) [reply packages/components/src/base-control/style.scss]: I guess the `.components-base-control .components-base-control__field` bottom padding makes this rule a bit redundunt. I'm actually ok removing it now.

Comment (youknowriad) [reply packages/components/src/base-control/style.scss]: Should we remove the rule from the table block too? (it's overridden there)

**Review by youknowriad — APPROVED** [review]

File: packages/components/src/base-control/style.scss (hunk 1 above)
Comment (jasmussen) [reply packages/components/src/base-control/style.scss]: I'll remove it there and ship. Thanks for fast review!

**Pull request merged**